src/Bicep.LangServer.IntegrationTests/DefinitionTests.cs

```diff
@@ -271,7 +312,7 @@ private static LocationLink ValidateDefinitionResponse(LocationOrLocationLinks r
 
         private static IEnumerable<object[]> GetData()
         {
-            return DataSets.NonStressDataSets.ToDynamicTestData();
+            return DataSets.NonStressDataSets.Where(x => x.Name == "LoadFunctions_CRLF").ToDynamicTestData();
```

Comment:
I'm assuming you meant to take this out!

Reply:
🤦 Fixed!